### ISSUE TITLE
Missing Factor 2 in python implementation of paraboloid (Trac #1364)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1356]
In paraboloid/releases/V01-07-00/python/fit_paraboloid.py

Nice docu in code. I followed the calculation of the formulars. 
There is an coefficient comparison between L. 30 and L. 38 which results in 
relation between coefficients in L. 41-43.
Here line 41 & 43 are missing a factor of 2.
L. 41 should be 
a = 2(A cosfi**2 + B sinfi**2)
and L 43 should be
c = 2(A sinfi**2 + B cosfi**2)

these factors are missing in all following calculations. 

I have done the exercise. The result should be:
case b=0 and a=c:
    A=B=a/2=c/2
case b=0 and a!=c:
    a=2A
    c=2B
case b!=0:
    tanfi_1/2 = 1/(2b) * [c-a +- sqrt( (a-c)**2 +4b**2)]  

Migrated from https://code.icecube.wisc.edu/ticket/1364
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "In paraboloid/releases/V01-07-00/python/fit_paraboloid.py\n\nNice docu in code. I followed the calculation of the formulars. \nThere is an coefficient comparison between L. 30 and L. 38 which results in \nrelation between coefficients in L. 41-43.\nHere line 41 & 43 are missing a factor of 2.\nL. 41 should be \na = 2(A cosfi**2 + B sinfi**2)\nand L 43 should be\nc = 2(A sinfi**2 + B cosfi**2)\n\nthese factors are missing in all following calculations. \n\nI have done the exercise. The result should be:\ncase b=0 and a=c:\n    A=B=a/2=c/2\ncase b=0 and a!=c:\n    a=2A\n    c=2B\ncase b!=0:\n    tanfi_1/2 = 1/(2b) * [c-a +- sqrt( (a-c)**2 +4b**2)]  ", 
    "reporter": "reimann", 
    "cc": "kmeagher@ulb.ac.be", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "Missing Factor 2 in python implementation of paraboloid", 
    "priority": "normal", 
    "keywords": "paraboloid", 
    "time": "2015-09-22T15:19:30", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151014T10:27:43: 
* kjmeagher changed owner from "" to "kjmeagher"
* kjmeagher changed status from "new" to "accepted"
* kjmeagher changed cc from "" to "kmeagher@ulb.ac.be"

**Comment by icecube-trac** [comment]
Trac update at 20151209T15:22:51: olivas changed milestone from "" to "IceRec V5"

**Comment by icecube-trac** [comment]
Trac update at 20160216T14:44:37: 
* kjmeagher commented:

> In changeset 141929:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="141929"
> fix math errors in python implementation of paraboloid
> fixes #1356
> ```


* kjmeagher changed status from "accepted" to "closed"
* kjmeagher changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
